advocacy_docs/partner_docs/DBeaverGuide/05-UsingDBeaver.mdx

```diff
@@ -0,0 +1,218 @@
+---
+title: 'Using DBeaver'
+description: 'Walkthroughs of example usage scenarios'
+---
+Once DBeaver is connected to an instance of EDB Postgres Advanced Server or EDB Postgres Extended, you can then access the capabilities of the integration demonstrated in the next section.
```

Comment:
Add PRO


Once DBeaver PRO is connected to an instance of EDB Postgres Advanced Server or EDB Postgres Extended, you can then access the capabilities of the integration.

remove demonstrated in the next section.